rasa/core/policies/ensemble.py

```diff
@@ -69,6 +70,15 @@ def __init__(
         self._check_priorities()
         self._check_for_important_policies()
 
+        self._rule_only_slots, self._rule_only_loops = self._get_rule_only_slots_loops()
+
+    def _get_rule_only_slots_loops(self) -> Tuple[List[Text], List[Text]]:
```

Comment:
I'd either add an `and` or make it a more general name: 
```suggestion
    def _get_rule_only_features(self) -> Tuple[List[Text], List[Text]]:
```

Reply:
added `and`